lib/visual_metrics.py

```diff
@@ -285,35 +288,51 @@ def ListOneTarget(codecs, rate, videofile, do_score, datatable,
     # Ignore results that score less than zero.
     if my_optimizer.Score(bestsofar) < 0.0:
       return
-    (datatable.setdefault(codec_name, {})
-             .setdefault(videofile.basename, [])
-             .append((bestsofar.result['bitrate'], bestsofar.result['psnr'])))
```

Comment:
Comment added. A type may make sense, but all that is done with it in Python is to print it as JSON.
(the full_results distinction will go away once everyone's agreed that the new format is better than the old one.)